tasks/build.py

```diff
@@ -12,11 +12,21 @@
 # Import invoke libs
 from invoke import task
 
+# Additional libs
+from shutil import which
+
 REPO_ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 TIMESTAMP_UI = ' -timestamp-ui' if 'DRONE' in os.environ else ''
 PACKER_TMP_DIR = os.path.join(REPO_ROOT, '.tmp', '{}')
 
 
+def _binary_install_check(binary):
+    '''Checks if the given binary is installed. Otherwise we exit with return code 10.'''
+    if not which(binary):
+        print("Couldn't find {}. Please install it to proceed.".format(binary))
```

Comment:
```python
exit_invoke(10, "Couldn't find {}. Please install it to proceed.", binary)
```

Reply:
Okay, I could have seen that. :laughing: Thanks for pointing out!